itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestTableLevelReplicationScenarios.java

```diff
@@ -263,12 +263,12 @@ public void testIncorrectTablePolicyInReplDump() throws Throwable {
     String[] replicatedTables = new String[] {};
     boolean failed;
     String[] invalidReplPolicies = new String[] {
-        primaryDbName + ".t1.t2", // Two explicit table names not allowed.
-        primaryDbName + ".['t1'].t2", // Table name and include list not allowed.
-        primaryDbName + ".t1.['t2']", // Table name and exclude list not allowed.
-        primaryDbName + ".[t1].t2", // Table name and include list not allowed.
-        primaryDbName + ".['t1+'].", // Abrubtly ended dot.
-        primaryDbName + "..[]" // Multiple dots
+        primaryDbName + ".t1.t2", // Didn't enclose table pattern within single quotes.
+        primaryDbName + ".'t1'.t2", // Table name and include list not allowed.
+        primaryDbName + ".t1.'t2'", // Table name and exclude list not allowed.
+        primaryDbName + ".'t1+'.", // Abrubtly ended dot.
```

Comment:
What is the behaviour for dbname.'t1'.'' and dbname.''? Empty include or exclude tables string.

Reply:
Noticed it fails now. Do we need to support it to dump only functions?

Reply:
We can add the empty include/exclude list in this test case.